sceptre/cli/list.py

```diff
@@ -89,6 +93,39 @@ def list_outputs(ctx, path, export):
                         ),
                         "text",
                     )
+
+    # Format outputs as !stack_output references.
+    elif export == "stackoutput":
+        for response in responses:
+            for stack_name, stack in response.items():
+                for output in stack:
+                    write(
+                        "!stack_output {0}.yaml::{1} [{2}]".format(
+                            stack_name,
+                            output.get("OutputKey"),
+                            output.get("OutputValue"),
+                        ),
+                        "text",
+                    )
+
+    # Format outputs as !stack_output_external references.
+    elif export == "stackoutputexternal":
```

Comment:
what are you thoughts about changing this to `stack_output_external`?

Reply:
See comments above.